documentation/c09-concepts.sil

```diff
@@ -122,6 +123,62 @@ Adding horizontal and vertical penalties to the typesetter’s queues is similar
 done with the \code{SILE.typesetter:pushPenalty(\{penalty = x\})} and
 \code{SILE.typesetter:pushVpenalty(\{penalty = y\})} methods.
 
+\subsection{Kerns}
+
+\define[command=SILEkern]{\font[family=Gentium Plus]{% Book Basic has no +smcp, but readers don't need to know, since we're only using Book Basic as a holdover from old SILE which did.
+S\lower[height=0.5ex]{I}L\kern[width=-.2em]\raise[height=0.6ex]{\font[features=+smcp]{e}}}}
+\define[command=SILEglue]{\font[family=Gentium Plus]{%
+S\lower[height=0.5ex]{I}L\glue[width=-.2em]\raise[height=0.6ex]{\font[features=+smcp]{e}}}}
+\code{\\kern}’s are a type of \code{\\glue}. The difference between the two is
+that, while a \code{\\glue} can be broken at the end of a line, a \code{\\kern}
+can’t be. Hearkening back to our \SILEkern example from the \em{Macros and
+Commands} chapter, consider that example, repeated enough times to cause a
+linebreak, but with \code{\\glue}’s everywhere \code{\\kern}’s are used
+instead:
+
+\line
+\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue%
+\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue%
+\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue%
+\SILEglue\SILEglue\color[color=#dd0000]{\SILEglue}\SILEglue\SILEglue%
+\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue%
+\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue%
+\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue%
+\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue\SILEglue%
+\line
+
+The 27\sup{th} of the 60 \SILEglue’s, in \color[color=#dd0000]{red}, is broken
+between its ‘L’ and ‘\raise[height=0.6ex]{\font[family=Gentium Plus,features=+smcp]{e}}’.
+Meanwhile, if we typeset the same line, except using \code{\\kern}’s as we had
+originally instead…
+
+\line
+\SILEkern\SILEkern\SILEkern\SILEkern\SILEkern\SILEkern\SILEkern\SILEkern%
```

Comment:
This is totally fine as a demonstration, but one thing I'd like to fix is getting rid of all the warnings when the manual builds. Right now we're ignoring the fact that even our own manual has known-bad behavior going on such as overflows we *don't* intend. Which brings up the idea what we may need a way to suppress warnings for things we *do* intend, such as this overflow. There is nothing in place now, but someday this will need some kind of suppress mechanism.